conjure-java-core/src/main/java/com/palantir/conjure/java/util/ErrorGenerationUtils.java

```diff
@@ -145,15 +176,27 @@ public static void addAllParametersWithSafetyAnnotationsToMethodBuilder(
 
     public static ParameterSpec buildParameterWithSafetyAnnotation(
             TypeMapper typeMapper, FieldDefinition argDefinition, boolean isSafe) {
-        return buildParameterWithSafetyAnnotationInternal(typeMapper, argDefinition, isSafe)
+        return buildParameterWithSafetyAnnotationInternal(typeMapper, argDefinition, isSafe, true)
                 .build();
     }
 
-    private static ParameterSpec.Builder buildParameterWithSafetyAnnotationInternal(
+    public static ParameterSpec buildUnsanitizedParameterWithSafetyAnnotationAndJsonProperty(
             TypeMapper typeMapper, FieldDefinition argDefinition, boolean isSafe) {
+        ParameterSpec.Builder parameterBuilder =
+                buildParameterWithSafetyAnnotationInternal(typeMapper, argDefinition, isSafe, false);
+        parameterBuilder.addAnnotation(AnnotationSpec.builder(JsonProperty.class)
+                .addMember("value", "$S", argDefinition.getFieldName().get())
+                .build());
+        return parameterBuilder.build();
+    }
```

Comment:
This should probably live next to `buildParameterWithSafetyAnnotationAndJsonProperty` and share more of the logic? Seems like the only difference is sanitized vs not (maybe we don't need _both_?)